inc/autoload/class-add-plugin-list.php

```diff
@@ -209,9 +210,13 @@ public function manage_plugins_custom_column( $column_name, $plugin_file, $plugi
 						$class = ' class="site-archived"';
 						$hint  = esc_attr__( ', Archived site', 'multisite-enhancements' );
 					}
+					elseif ( $this->is_deleted( $key ) ) {
+						$class = ' class="site-deleted"';
+						$hint  = esc_attr__( ', Deleted site', 'multisite-enhancements' );
```

Comment:
The comma on start is hard for translators, so please add in minimum the hint for the translators. See https://gist.github.com/bueltge/49c652187ef10a448bfb#you-want-to-output-the-username-in-a-sentence

Reply:
I would rather leave the comma out of the translated string, like
```
$hint  = ', ' . esc_attr__( 'Archived site', 'multisite-enhancements' );
```
this way the structure won't be broken by a translator mistake.

And how about we use the native WP translations for archived and deleted, so they remain consistent with other areas of the admin dashboard (screenshot below). What do you think?

![image](https://user-images.githubusercontent.com/1033735/69804588-ae5c5800-11bd-11ea-952a-cc69d4fe42a2.png)

While I was testing it, I realized a site can be both deleted and archived, so I'll change that `elseif` statement.

Reply:
The native version would be really fine, god idea. So we need only the comma outside the string for translation. However, I'm not sure there are languages there always use the comma for the arrangement of strings.
Thanks also for the hints, good part of testing.

Reply:
Cool, I'll commit these changes. WP already uses the comma to separate site statuses in the site list, so I think it's ok.